[2022-10-26 14:30:02,670] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag.fetch_file scheduled__2022-10-26T14:15:00+00:00 [queued]>
[2022-10-26 14:30:02,678] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag.fetch_file scheduled__2022-10-26T14:15:00+00:00 [queued]>
[2022-10-26 14:30:02,678] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-26 14:30:02,678] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-10-26 14:30:02,679] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-26 14:30:02,691] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): fetch_file> on 2022-10-26 14:15:00+00:00
[2022-10-26 14:30:02,693] {standard_task_runner.py:52} INFO - Started process 4794 to run task
[2022-10-26 14:30:02,696] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dag', 'fetch_file', 'scheduled__2022-10-26T14:15:00+00:00', '--job-id', '140', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpnr0i_mo7', '--error-file', '/tmp/tmpu1t5zpvh']
[2022-10-26 14:30:02,697] {standard_task_runner.py:80} INFO - Job 140: Subtask fetch_file
[2022-10-26 14:30:02,753] {task_command.py:369} INFO - Running <TaskInstance: dag.fetch_file scheduled__2022-10-26T14:15:00+00:00 [running]> on host cf01c04ee00d
[2022-10-26 14:30:02,821] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dag
AIRFLOW_CTX_TASK_ID=fetch_file
AIRFLOW_CTX_EXECUTION_DATE=2022-10-26T14:15:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-26T14:15:00+00:00
[2022-10-26 14:30:05,084] {python.py:173} INFO - Done. Returned value was: None
[2022-10-26 14:30:05,098] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dag, task_id=fetch_file, execution_date=20221026T141500, start_date=20221026T143002, end_date=20221026T143005
[2022-10-26 14:30:05,278] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-26 14:30:05,315] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
